.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+sudo: false
+
+python:
+  - "3.5"
+  - "2.7"
+
+install:
+  # Travis needs to update pip/setuptools
+  - pip install --upgrade setuptools pip
+  # install test plugins
+  - pip install --upgrade pytest pytest-check-links
+  - pip install https://github.com/computationalmodelling/nbval/archive/master.zip
+  # install requirements
+  - pip install -r requirements.txt
+
+script:
+  - pytest --nbval --check-links .
```

Comment:
Do you want to use `--nbval-lax`? To my mind, this (checking that notebooks run, and then checking output on specified cells) is a better starting point than checking all output.